resources/lib/addon.py

```diff
@@ -333,8 +333,24 @@ def iptv_epg():
 @plugin.route('/update/repos')
 def update_repos():
     """Force an update of the repositories"""
-    execute_builtin('UpdateAddonRepos')
-    ok_dialog(heading=localize(30450), message=localize(30451))
+    if get_global_setting('general.addonupdates') == 0:  # Automatic updates is enabled
+        execute_builtin('UpdateAddonRepos')
+        ok_dialog(heading=localize(30450), message=localize(30451))  # Repositories are being updated
+    else:
+        ok_dialog(heading=localize(30452), message=localize(30453))  # Automatic updates is disabled
+        show_settings_addons()
+
+
+@plugin.route('/show/settings/addons')
+def show_settings_addons():
+    """Open the Kodi System Settings in the Add-ons category"""
+    execute_builtin('ActivateWindow(SystemSettings,addons)')
+
+
+@plugin.route('/open/information')
+def show_information():
+    """Show the Add-on information pane"""
+    execute_builtin('ActivateWindow(SystemSettings,addons)')
```

Comment:
I left this in for someone to try, but since it is not functional we might as wel remove it.